services/sandbox-runtime/Dockerfile

```
@@ -134,6 +134,48 @@ RUN set -eux; \
+# (SANDBOX_DOCKER_IN_CONTAINER + a sysbox/kata tier): the binaries are not
+# setuid and cannot start a daemon without caps the one-shot/non-DinD paths
+# drop, so adding them does not change the one-shot (uid 65534) attack surface.
+# The session entrypoint starts an inner dockerd only in DinD mode.
+#   docker-ce/cli + compose/buildx plugins + containerd = the engine
+#   tini       = PID-1 reaper (DinD spawns many short-lived containerd-shims)
+#   uidmap     = newuidmap/newgidmap (rootless/userns helpers)
+#   fuse-overlayfs + iptables = inner daemon storage + networking fence
+#   util-linux = setpriv, to drop dockerd's root back to uid 10001 for runnerd
+# Versions are the live Docker apt-repo stable channel; pin here if a build
+# needs to be byte-reproducible.
+RUN install -m 0755 -d /etc/apt/keyrings \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)